pkg/deployer/bluegreen.go

```diff
@@ -31,12 +31,23 @@ func NewBlueGrean(mode string, logger *Logger.Logger, awsConfig schemas.AWSConfi
 			PrevInstanceCount: map[string]schemas.Capacity{},
 			PrevVersions:      map[string][]int{},
 			Stack:             stack,
+			StepStatus: map[int64]bool{
+				tool.STEP1: false,
```

Comment:
Is is possible to change each step names with more meaningful names?
ex) tools.DEPLOY_STEP, tools.CHECK_PREVIOUS_STEP 
I think it would be better for writing and reading the step names.

Reply:
@asbubam  I fixed it. :) Thanks

Reply:
Thank you for applying.